packages/terra-popup/src/terra-dev-site/doc/example/PopupStandard.jsx

```diff
@@ -1,6 +1,40 @@
-import React from 'react';
-import PopupCommonExample from 'terra-popup/lib/terra-dev-site/doc/common/PopupCommonExample';
+import React, { useRef, useState } from 'react';
+import Button from 'terra-button';
+import Popup from 'terra-popup';
+import Placeholder from 'terra-doc-template/lib/Placeholder';
 
-const PopupStandard = () => <PopupCommonExample title="Default Popup" isArrowDisplayed={false} />;
+function PopupStandard() {
+  const buttonElement = useRef(null);
+  const [open, setOpen] = useState(false);
+
+  const setButtonNode = (node) => {
+    buttonElement.current = node;
+  };
+
+  const getButtonNode = () => buttonElement.current;
+
+  const handleButtonClick = () => {
+    setOpen(true);
+  };
+
+  const handleRequestClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <React.Fragment>
+      <Button text="Default Popup" onClick={handleButtonClick} refCallback={setButtonNode} />
+      <Popup
+        attachmentBehavior="auto"
```

Comment:
Shouldn't need attachmentBehavior or contentAttachment values set in the default example.

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/927/commits/f26bebb7c61ac4b5747385bd2a04d26a4785fb3d.